core/pom.xml

```diff
@@ -50,6 +129,11 @@
       <artifactId>system-stubs-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>uk.org.webcompere</groupId>
+      <artifactId>system-stubs-core</artifactId>
+      <scope>test</scope>
```

Comment:
Not a fan of having this dependency. If mocking `System.getenv` the single reason to have this, then maybe we should make this `UnaryOperator<String>` customizable for tests.